examples/typescript/create-preauth-req.ts

```diff
@@ -77,7 +78,7 @@ client.region = Region.US_PHOENIX_1;
         // create pre authenticated request to generate the url
         const resp = await client.createPreauthenticatedRequest(createPreauthenticatedRequest);
         const baseUrl =
-          "https://objectstorage." + common.Region.US_PHOENIX_1.regionId + ".oraclecloud.com";
+          "https://" + hostName + "." + common.Region.US_PHOENIX_1.regionId + ".oraclecloud.com";
         const downloadUrl = resp.preauthenticatedRequest.accessUri;
```

Comment:
Here also... 

> Can the value ".oraclecloud.com" be replaced by common.Realm.OC1.secondLevelDomain here and also hostName variable with serviceName?

Reply:
and maybe use templated strings `${serviceName}.${common.Region.US_PHOENIX_1.regionId}.${common.Realm.OC1.secondLevelDomain}`